components/clubs-page.jsx

```diff
@@ -208,7 +166,29 @@ var ClubsPage = React.createClass({
           <Quote/>
         </section>
         <section>
-          <IconLinks/>
+          <IconLinks links={[
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-curriculum.svg",
+              imgAlt: "icon curriculum",
+              head: "Curriculum",
+              subhead: "Modular Web Literacy curriculum"
+            },
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-connect.svg",
+              imgAlt: "icon connect",
+              head: "Connect",
+              subhead: "Connect with other Club Leaders"
+            },
+            {
+              linkTo: "fixme",
+              imgSrc: "/img/icon-tips.svg",
+              imgAlt: "icon tips",
+              head: "Helpful Tips",
+              subhead: "Tips for running your Club"
+            }
+          ]}/>
```

Comment:
Hmm, I definitely like the direction you're going here, but this code seems to suggest yet another iteration that looks something like this:

```jsx
<IconLinks>
  <IconLink linkTo="fixme" imgSrc="/img/icon-connect.svg"
   imgAlt="icon connect" head="Connect" subhead="Connect with other Club Leaders"/>
  ...
</IconLinks>
```

And so on? It definitely *feels* more React-ish, and it would also allow us to define a more granular `propTypes` on `IconLink` that will let us catch errors and self-document our code a bit better.

What do you think?